src/App.js

```diff
@@ -9,94 +9,97 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      areGamesLoaded: false,
-      areGenresLoaded: false,
-      error: null,
+      errors: null,
       games: null,
       genres: null,
       popUpInfo: null,
-      popUpGenreID: null,
+      popUpGenre: null,
       searching: '',
       filteredGames: []
     }
   }
+  getData = (request) => {
```

Comment:
refactored our two fetch calls into a function. I am not sure what to name this function(was thinking about fetch lol). getData is likely way to generic.

Reply:
Haha, fetchData would work for me! Another great refactor, good idea Justin.